data/tekton-pipelines/windows-efi-installer-pipeline.yaml

```diff
@@ -44,6 +44,11 @@ spec:
       description: Name of Windows ISO datavolume
       name: isoDVName
       type: string
+    - default: windows.11.virtio
+      description: Name of the preference which will be used for this DV by default.
+      name: defaultPreferenceLabel
```

Comment:
preferenceName parameter is used for information about VM which is created to install the iso. The defaultPreferenceLabel is used for result DV. These two parameters might be different. E.g. for installation you want to use windows-11-some-fency-feature-enabled, but the result you want to use with different preference (e.g. windows-11-regular-user-disk)

Reply:
I agree that the two parameters are technically different, but it might be simple and intuitive to the user, if the DV would reflect the instancetype and preference which were used to create the DV. Avoiding the additional parameter would also keep the "surface" to the user smaller. Since this pipeline targets users who starts learning about kubevirt-tekton-tasks, I would vote to keep it as simple as possible, even at the price of flexibility.